### PR TITLE
[TS] LPS-190994

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:clustering

**Comment by Kvale1733** [comment]
@marianoalvarosaiz No regressions found. Testing passed following the ticket steps.

**Comment by tinatian** [comment]
@marianoalvarosaiz For https://github.com/liferay-core-infra/liferay-portal/pull/5356/commits/6e6172964abfa70b85fdeaa5f78e7106bb7ada81, please create a separate ticket to remove it, and you also need to provide upgrade process to clean this configuration from database.

Thanks.
Tina

File: modules/apps/portal-scheduler/portal-scheduler/src/main/java/com/liferay/portal/scheduler/internal/SchedulerEngineHelperImpl.java [hunk 1]
```diff
@@ -547,13 +548,24 @@ public SchedulerJobConfiguration addingService(
 			Trigger trigger = null;
 
 			if (Validator.isNotNull(triggerConfiguration.getCronExpression())) {
+				if (StringUtil.equals(
+						triggerConfiguration.getCronExpression(),
+						StringPool.BLANK)) {
+
+					return null;
+				}
+
```
Comment (tinatian) [inline modules/apps/portal-scheduler/portal-scheduler/src/main/java/com/liferay/portal/scheduler/internal/SchedulerEngineHelperImpl.java:573]: @marianoalvarosaiz This part of code is dead code, you will never reach to this as `if (Validator.isNotNull(triggerConfiguration.getCronExpression()))` also checks empty string.

Please also take a look at your test why it did not catch this.

Thanks.
Tina.

**Comment by tinatian** [comment]
@marianoalvarosaiz Please see my comment and resend.

Thanks.
Tina.

**Comment by marianoalvarosaiz** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/5394

**Pull request closed without merging**